### ISSUE TITLE
Pressing space no longer toggles message selection in the message list

### ISSUE THREAD

**Issue by Simon818** [UnigramDev/Unigram#2727]
Unigram 8.7 (7363) X64 Direct

**Describe the bug**
In previous Unigram betas, one could turn on message selection mode from the right-click menu, arrow up and down the message list, and press space to select or unselect messages. This no longer works since the message selectors are now checkboxes. You can tab from the message list and reach the checkbox for the currently-focused message, but this is an extra step. It also means screen-readers will not read the selected (or unselected) state of a message when arrowing up and down the list.
**To Reproduce**
1. Open any conversation with 2 or more messages in it.
2. Turn on selection mode (right-click and choose "select")
3. tab or shift+tab to the list of messages.
4. Use the arrow keys to navigate to a message other than the one that is already selected.
5. Press the space bar.
6. Press the tab key, then press space again.
**Expected behavior**
In previous Unigram versions, pressing the space bar in step 5 should have selected the message. Instead, you needed to tab to the checkbox for that message and press space on it instead (step 6.)

**Comment by Simon818** [comment]
Would super appreciate this being looked into. Selecting messages with a keyboard is now a confusing mess of tab, space, shift+tab, arrow, repeat. This is a clear regression from previous Unigram versions.

**Comment by Simon818** [comment]
This has been fixed as of 8.8 (7440) if not earlier. Now the list item *is* the checkbox, so I can directly arrow/space through them. I'm not sure who was responsible for that, but thank you for solving it in such a neat way.
